src/index.html

```diff
@@ -0,0 +1,240 @@
+             </li>
+             <li>
+                <a href="#anchor-socials">Socials</a>
+             </li>
+          </ul>
+       </nav>
+    </div>
+ </header>
+ <div class="container-fluid">
+ <div class="row robot-header white">
+       <div class="robot-header-left col-lg-6 col-md-12 col-sm-12 col-xs-12">
+          <h1>Baumeister</h1>
+          <div class="robot-flying">
+             <img class="hover_wide" src="assets/img/Baumeister-Roboter.png" alt="Baumeister-Roboter">
+          </div>
+          <div class="robot-burst">
+             <img class="big_boost hover_short1" src="assets/img/big_boost.png" alt="">
+             <img class="middle_boost hover_short2" src="assets/img/middle_boost.png" alt="">
+             <img class="small_boost hover_short3" src="assets/img/small_boost.png" alt="">
+          </div>
+       </div>
+       <div class="robot-header-right col-lg-6 col-md-12 col-sm-12 col-xs-12">
+          <h1>The frontend build workflow for your needs</h1>
+          <p>
+             The aim of this project is to help you to build your things.
+             <br>
+             From Bootstrap themes over static websites to single page applications.
+          </p>
+          <div><a class="button" href="#anchor-install" >Install Now</a></div>
+    </div>
+ </div>
+ <div class="row our-patners gray">
+       <div class="patner bmbf col-lg-2 offset-lg-2 col-md-2 offset-md-1 col-ms-12 col-xs-12">
+          <img src="assets/img/bmbf.svg" alt="bmbf" class="img-circle img-responsive center">
+       </div>
+       <div class="patner micromata offset-lg-1 offset-md-2 col-lg-2 col-md-2 col-ms-12 col-xs-12">
+          <img src="assets/img/MICROMATA_pfeile_gross_RGB.svg" alt="Micromata" class="img-circle img-responsive center">
+       </div>
+       <div class="patner offset-lg-1 offset-md-1 col-lg-2 col-md-2 col-ms-12 col-xs-12">
+          <img src="assets/img/DLR.png" alt="DLR">
+       </div>
+ </div>
+ <div class="row">
+    <div id="anchor-about" class="about white col-lg-12 col-md-12 col-ms-12 col-xs-12">
+       <h2>What advantages does Baumeister offer?</h2>
+       <p>
+          The administration of dependencies can be completely solved via the `package.json` of the project. No additional file needs to be modified, e.g. if new libraries or files like fonts need to be added to the JavaScript and CSS bundles. The package.json section `baumeister' contains the properties `bundleCSS`, `bundleExternalJS` and `includeStaticFiles` for this purpose.
+          <br><br>
+          Another advantage of Baumeister over other solutions is the simple adaptation of automation tasks through the maintainable structure. Each task is a separate ES6 module and in the Gulpfile these tasks are only plugged together. If a task is not needed during the entire automation process, it can therefore be localized very quickly and easily deactivated - even if you are not familiar with Baumeister's overall structure.
```

Comment:
> Another advantage of Baumeister over other solutions is the simple adaptation of automation tasks through the maintainable structure. Each task is a separate ES6 module and in the Gulpfile these tasks are only plugged together. If a task is not needed during the entire automation process, it can therefore be localized very quickly and easily deactivated - even if you are not familiar with Baumeister's overall structure.


ersetzen durch:

> Another advantage of Baumeister over other solutions is the simple adaptation of automation tasks through the maintainable structure. Each task is a separate npm run script and in the package.json these tasks are only plugged together. If a task is not needed during the entire automation process, it can therefore be localized very quickly and easily deactivated - even if you are not familiar with Baumeister's overall structure.

Reply:
Done